monitor.py

```diff
@@ -866,17 +867,21 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
                     {self.canonicalise(x['Name']): x['Count'] for x in clean_data}
                 )
 
-            elif event_type == 'Backpack':
+            elif event_type == 'BackPack':
                 # TODO: v31 doc says this is`backpack.json` ... but Howard Chalkley
                 #       said it's `Backpack.json`
                 with open(join(self.currentdir, 'Backpack.json'), 'rb') as backpack:  # type: ignore
                     try:
-                        entry = json.load(backpack)
+                        # Preserve property order because why not?
+                        entry = json.load(backpack, object_pairs_hook=OrderedDict)
```

Comment:
This should be ordered anyway as normal python dicts are ordered now

Reply:
I was cribbing from the equivalent in CargoJSON code.

Reply:
Ah okay